index.src.html

```diff
@@ -91,11 +92,10 @@
       text: creation URL
     urlPrefix: workers.html
       text: running a worker; url: run a worker
-    urlPrefix: dom.html
-      text: document referrer policy; url: concept-document-referrer-policy; for: Document
   type: interface
     urlPrefix: dom.html
       text: Document
+      text: referrer policy; url: concept-document-referrer-policy; for: Document
```

Comment:
I think it should be under the dfns, but with

```
    urlPrefix: dom.html
      text: referrer policy; url: concept-document-referrer-policy; for: Document
```

and then you can do `<a for="Document">referrer policy</a>`.

If that doesn't work we likely have a Bikeshed bug on our hands :(